### ISSUE TITLE
New problem with 'entries' in property name

### ISSUE THREAD

**Issue by aloysiustany** [swagger-api/swagger-ui#6033]
### Q&A (please complete the following information)
 - OS: macOS
 - Browser: chrome
 - Version: 81
 - Method of installation: editor.swagger.com
 - Swagger-UI version: I think editor.swagger.com is using the latest i.e. 3.25.4 
 - Swagger/OpenAPI version: Swagger 2.0

### Content & configuration

This looks like another new problem when having 'entries' as property name. Other properties in the same level of 'entries' properties, fails to load with error "Could not render this component, see the console"

Example Swagger/OpenAPI definition:
```yaml
  swagger: '2.0'
  info:
    title: Test Me
    version: '1'
  paths:
    '/test/me':
      get:
        summary: Test api
        operationId: getTestMe
        produces:
          - '*/*'
        responses:
          '200':
            description: OK
            schema:
              $ref: '#/definitions/TestMeDto'
          '500':
            description: Internal Server Error
        deprecated: false
  definitions:
    TestMeDto:
      type: object
      properties:
        name: 
          type: string
        age: 
          type: string
        entries: 
          type: array
          items:
            $ref: '#/definitions/TestMeEntries'
    TestMeEntries:
      type: object
      properties:
        date: 
          type: string
        time: 
          type: string

```

### To reproduce...

Try to load swagger ui and use yaml above

### Screenshots
![image](https://user-images.githubusercontent.com/2670144/82844100-0be31000-9e94-11ea-8a6e-7fca6affdd66.png)

if I change entries to entries-new, it works as expected:
![image](https://user-images.githubusercontent.com/2670144/82844149-477dda00-9e94-11ea-9fe1-63c18d65268d.png)

**Comment by shindigira** [comment]
I am confirming this issue where any property named `entries` causes an error. In my case where I am utilizing `hapi-swagger` which in turn relies on `swagger-ui`, I get the error below.

<img width="1665" alt="Screen Shot 2020-05-26 at 4 17 58 PM" src="https://user-images.githubusercontent.com/13324863/82946692-cfcfae00-9f6c-11ea-8e14-b129beda8ba6.png">

Note: I am not **missing valid version field** despite the presented error message. This goes away when I rename "entries" to any other field name.

**Comment by tim-lai** [comment]
@aloysiustany @shindigira this will be fixed in the next release, via bec6aeb
